onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc

```diff
@@ -1478,6 +1547,28 @@ TEST(ReductionOpTest, ArgMax_int32) {
   test.Run();
 }
 
+TEST(ReductionOpTest, ArgMax_int32_last_index) {
+  OpTester test("ArgMax", 12);
+  test.AddAttribute("axis", (int64_t)1);
+  test.AddAttribute("keepdims", (int64_t)1);
+  test.AddAttribute("select_last_index", (int64_t)1);
+
+  test.AddInput<int32_t>("data", {3, 2, 2},
+                         {2, 2,
+                          3, 4,
+
+                          5, 6,
+                          7, 8,
+
+                          10, 10,
+                          11, 12});
+  test.AddOutput<int64_t>("reduced", {3, 1, 2},
+                          {1, 1,
+                           1, 1,
+                           1, 1});
```

Comment:
Would be slightly better if the correct value for each reduction wasn't always in the same location.

Reply:
Will add a new test. This one checks the the results are the same given there are not duplicates.

---
In reply to: [400579647](https://github.com/microsoft/onnxruntime/pull/3341#discussion_r400579647) [](ancestors = 400579647)